helios2.code-workspace

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
New commit (or _prob_ ditch this)?

Reply:
Whoops, yeah, I thought I had gotten rid of that